Hivemind/Assets/Scripts/ResourceNode.cs

```diff
@@ -29,12 +29,23 @@ private void Awake()
         resourceAmount = BaseResourceAmount;
         futureResourceAmount = resourceAmount;
         myResourceNode = gameObject;
-        resourceNodeTransform = gameObject.transform; GameWorld.AddNewResource(this);
+        resourceNodeTransform = gameObject.transform;
+        myResourceNode.GetComponent<MeshRenderer>().enabled = false;
+        GameWorld.AddNewResource(this);
+    }
+
+    public void AddToKnownResourceList()
```

Comment:
moet dit niet een functie zijn binnen de GameWorld?

Reply:
Deze functie veranderd ook de meshrenderer en roept daarna de methode in gameworld aan om hem toe te voegen. Zo hoeven ants niet de gameworld aan te roepen maar alleen hun target